ui-tests/src/test/java/org/openmrs/reference/page/SystemAdministrationPage.java

```diff
@@ -36,4 +42,14 @@ public StylesGuidePage clickOnStylesGuideAppLink() {
         clickOn(STYLES_GUIDE_LINK);
         return new StylesGuidePage(this);
     }
+    
+    public  AdministrationPage activateWeekends(String value) {
+    	clickOn(ADMINISTRATION);
+    	clickOn(MANAGE_GLOBALS);
+    	driver.findElement(EDIT_WEEKENDS).click();
```

Comment:
please check if the value isn't active already in order to set it to that else don't re-active it and if it was initially found active, ensure it's not set even in the before method